network/protocol.go

```diff
@@ -271,12 +335,13 @@ func (p *BzzPeer) ID() enode.ID {
 * Version: 8 byte integer version of the protocol
 * NetworkID: 8 byte integer network identifier
 * Addr: the address advertised by the node including underlay and overlay connecctions
+* Capabilities: the capabilities bitvector
 */
 type HandshakeMsg struct {
-	Version   uint64
-	NetworkID uint64
-	Addr      *BzzAddr
-	LightNode bool
+	Version      uint64
+	NetworkID    uint64
+	Addr         *BzzAddr
+	Capabilities Capabilities
```

Comment:
The last time I added the `LightNode` flag I had to bump the protocol version in `BzzSpec`. Maybe that changed since then but I could not find it in this PR.

Reply:
Yes! Important! Thanks!